lib/public/views/QcFlags/details/qcFlagDetailsComponent.js

```diff
@@ -106,6 +102,17 @@ export const qcFlagDetailsComponenet = (qcFlagDetailsModel, additionalDetailsCon
         }
     };
 
+    /**
+     * Function called when QC flag post verification button is clicked
+     * @return {void}
+     */
+    const qcFlagPostVerificationOnclick = () => {
```

Comment:
qcFlagVerificationSendHandler

=> onclick is the name of the parameter, handler is the actual argument
In the context of the function is what happens on click, outside it's the function that handle the sending. It can (and should normally...) be triggered also on different events, such as form send (pressing enter key)

Reply:
@graduta opinion?

Reply:
To add more, a function name should never indicate how or where it should be used, but only what it does. Here what it does is handle the sending of verification, onclick is an indication on when to use it which is wrong

Reply:
I think @martinboulais 's explanation is well writte and there is no need for me to express an opinion as I have nothing else to add.
I agree with his recommendation. This is emphasized on multiple conventions readings if you would like to look a bit more into details

Reply:
renamed